ocaml/xapi/workload_balancing.ml

```diff
@@ -114,6 +115,66 @@ let assert_wlb_enabled ~__context =
   if not (Db.Pool.get_wlb_enabled ~__context ~self:pool) then
     raise_disabled ()
 
+let read_file path =
```

Comment:
this does not only `read_file`, but read and destroy the tmp file, 
deserver a better name?
BTW, this functions can be put to some common place (if not have similar already)?

Reply:
Maybe `read_and_clean_file `  or `read_and_remove_file`